lib/cylc/gui/gscan.py

```diff
@@ -981,6 +1035,10 @@ def update(self):
             )
             title = suite_info.get("title")
 
+            if 'tasks-by-state' in suite_info:
+                self.tasks_by_state[suite + host] = suite_info[
```

Comment:
Probably better to use (suite, host) as the key for `tasks_by_state`, as elsewhere.